managed/CounterStrikeSharp.API/Core/Memory/MemoryManager.cs

```diff
@@ -0,0 +1,318 @@
+        }
+
+        [RequiresPermissions("@css/generic")]
+        private void OnCommand(CCSPlayerController? caller, CommandInfo info)
+        {
+            switch (info.GetArg(1))
+            {
+                case "stats":
+                {
+                    PrintStatistics(info);
+                } break;
+
+                case "start":
+                {
+                    Start();
+                } break;
+
+                case "stop":
+                {
+                    Stop(true);
+                } break;
+
+                case "pause":
+                {
+                    Stop(false);
+                }  break;
+
+                case "resume":
+                {
+                    Resume();
+                } break;
+
+                default:
+                {
+                    info.ReplyToCommand("Valid usage: css_memory [option]\n" +
+                                               "  stats - Print garbage collector statistics.\n" +
+                                               "  start - Starts the memory manager that handles leaking resources.\n" +
+                                               "  stop - Stops the memory manager.\n" +
+                                               "  pause - Stops the memory manager.\n" +
+                                               "  resume - Resumes the memory manager.\n");
+                } break;
+            }
+        }
+
+        public void Load()
+        {
+            _commandManager.RegisterCommand(new("css_memory", "Counter-Strike Sharp Memory Manager options.",
+                OnCommand)
+            {
+                ExecutableBy = CommandUsage.CLIENT_AND_SERVER,
+                MinArgs = 1,
+                UsageHint = "[option]\n" +
+                            "  stats - Print garbage collector statistics.\n" +
+                            "  start - Starts the memory manager that handles leaking resources.\n" +
+                            "  stop - Stops the memory manager.\n" +
+                            "  pause - Stops the memory manager.\n" +
+                            "  resume - Resumes the memory manager.\n",
+            });
+
+            Start();
+        }
+
+        private void PrintStatistics(CommandInfo info)
+        {
+            info.ReplyToCommand("Memory Manager Statistics:\n" +
+                                      $"State: {State}\n" +
+                                      $"Total Released: {TotalReleased}\n" +
+                                      $"Last Released: {LastReleased}\n" +
+                                      $"Current Resources: {CurrentResources}\n" +
+                                      $"Last Updated: {LastUpdated.ToString("yyyy.MM.dd hh:mm:ss tt")} (UTC)\n" +
+                                      $"Heap Memory Usage: ~{TotalMemoryMB:F5} MB ({TotalMemory} bytes)\n" +
+                                      $"Total Allocated Bytes: ~{TotalAllocatedMB:F5} MB ({TotalAllocated} bytes)");
```

Comment:
not sure if we should call those GC methods multiple times, I'm down for any change